### ISSUE TITLE
how to inject user root to kali image run standalone?

### ISSUE THREAD

**Issue by mjakubowskilkr** [kasmtech/KasmVNC#91]
I need to add user root to Kali container. Can I add any parameter to command `sudo docker run --rm  -it --shm-size=512m -p 6901:6901 -e VNC_PW=password kasmweb/<image>:<tag>` ?

**Comment by j-travis** [comment]
Yes, you are free to add any docker run flag you want , including `--user root`
https://docs.docker.com/engine/reference/commandline/run/
